apps/prairielearn/src/pages/instructorFileTransfer/instructorFileTransfer.ts

```diff
@@ -46,41 +71,60 @@ router.get(
       res.locals.user.user_id,
     );
     const from_course = await selectCourseById(file_transfer.from_course_id);
-    // Split the full path and grab everything after questions/ to get the QID
-    const question_exploded = path.normalize(file_transfer.from_filename).split(path.sep);
-    const questions_dir_idx = question_exploded.findIndex((x) => x === 'questions');
-    const qid = question_exploded.slice(questions_dir_idx + 1).join(path.sep);
-    const editor = new QuestionTransferEditor({
-      locals: res.locals as any,
-      from_qid: qid,
-      from_course_short_name: from_course.short_name,
-      from_path: path.join(config.filesRoot, file_transfer.storage_filename),
-    });
-    const serverJob = await editor.prepareServerJob();
-    try {
-      await editor.executeWithServerJob(serverJob);
-    } catch {
-      res.redirect(res.locals.urlPrefix + '/edit_error/' + serverJob.jobSequenceId);
-      return;
-    }
 
-    debug('Soft-delete file transfer');
-    await sqldb.queryOneRowAsync(sql.soft_delete_file_transfer, {
-      id: req.params.file_transfer_id,
-      user_id: res.locals.user.user_id,
-    });
-    debug(`Get question_id from uuid=${editor.uuid} with course_id=${res.locals.course.id}`);
-    const question = await selectQuestionByUuid({
-      // TODO: we'll have to change something here once we allow instructors to
-      // copy questions that have been shared with their course.
```

Comment:
Was this TODO addressed? I'm not sure exactly what we thought had to be changed here, but I don't see any change in the new code here.